docs/frontend_utilisation_guide.md

```diff
@@ -1,15 +1,15 @@
 # Utilising the Frontend
 ## Overview 
-The Scalable Pixel Streaming Frontend is a library of HTML, CSS and TypeScript code that runs in client web browsers to help users connect to Scalable Pixel Streaming applications and interact with them. It is able to achieve this by consuming the Pixel Streaming Frontend and UI libraries and by extending their signalling server and websocket packages the Pixel Streaming Fronted can be configured to work with Scalable Pixel Streaming signalling severs.   
+The Scalable Pixel Streaming Frontend is a library of HTML, CSS and TypeScript code that runs in client web browsers to help users connect to Scalable Pixel Streaming applications and interact with them. It is able to achieve this by consuming the Pixel Streaming Frontend and UI libraries and by extending their signalling server and WebSocket packages the Pixel Streaming Fronted can be configured to work with Scalable Pixel Streaming signalling severs.   
```

Comment:
Fix:
> Pixel Streaming Fronted

to:
> Pixel Streaming Frontend

Reply:
Resolved in 2b0b3dd0bba4e3fb340e31d078b790f45db9ae49